plugin/src/main/java/io/jenkins/plugins/coverage/model/CoverageViewModel.java

```diff
@@ -558,39 +558,32 @@ public DetailedColumnDefinition getLoc() {
             return new DetailedColumnDefinition(Messages.Coverage_Not_Available(), "0");
         }
 
-        protected String printCoverage(final Coverage coverage) {
-            if (coverage.isSet()) {
-                return coverage.formatCoveredPercentage(browserLocale);
-            }
-            return Messages.Coverage_Not_Available();
-        }
-
         /**
          * Creates a table cell which colorizes the shown coverage dependent on the coverage percentage.
          *
          * @param coverage
-         *         The coverage as percentage
-         * @param text
-         *         The text to be shown which represents the coverage
+         *         the coverage of the element
          * @param tooltip
-         *         The tooltip which describes the value
+         *         the tooltip which describes the value
          *
          * @return the create {@link DetailedColumnDefinition}
          */
-        protected DetailedColumnDefinition createColoredCoverageColumn(final CoveragePercentage coverage,
-                final String text, final String tooltip) {
-            double percentage = coverage.getDoubleValue();
-            String sort = String.valueOf(percentage);
-            DisplayColors colors = CoverageLevel.getDisplayColorsOfCoverageLevel(percentage, COLOR_PROVIDER);
-            String tag = span()
-                    .withTitle(tooltip)
-                    .withStyle(String.format(
-                            "color:%s; background-image: linear-gradient(90deg, %s %f%%, transparent %f%%); display:block;",
-                            colors.getLineColorAsHex(), colors.getFillColorAsHex(),
-                            percentage, percentage))
-                    .withText(text)
-                    .render();
-            return new DetailedColumnDefinition(tag, sort);
+        protected DetailedColumnDefinition createColoredCoverageColumn(final Coverage coverage, final String tooltip) {
+            double percentage = coverage.getCoveredPercentage().getDoubleValue();
+            if (coverage.isSet()) {
+                DisplayColors colors = CoverageLevel.getDisplayColorsOfCoverageLevel(percentage, COLOR_PROVIDER);
+                String cell = div().withClasses(COVERAGE_COLUMN_OUTER).with(
+                                div().withClasses(COVERAGE_COLUMN_INNER)
+                                        .withStyle(String.format(
+                                                "color:%s; background-image: linear-gradient(90deg, %s %f%%, transparent %f%%);",
```

Comment:
I think here we have the same problem. The whole UI looks somewhat funny in dark mode (I need to update the datatables CSS as well)

![Bildschirmfoto 2022-05-22 um 22 36 02](https://user-images.githubusercontent.com/503338/169714799-79abff3f-5c3a-4f9a-b3e0-136ddf2780f8.png)